? .sourcedirs
? .uts_version
? .config
? .hdepend
? .depend
? modules
? patch
? patch.tim
? adv_pci1710.diff
? comedi/.depend
? comedi/.comedi_fops.o.flags
? comedi/.proc.o.flags
? comedi/.range.o.flags
? comedi/.drivers.o.flags
? comedi/.kvmem.o.flags
? comedi/.rt.o.flags
? comedi/.comedi_ksyms.o.flags
? comedi/drivers/.depend
? comedi/drivers/.8255.o.flags
? comedi/drivers/.mite.o.flags
? comedi/drivers/.adl_pci9118.o.flags
? comedi/drivers/.cb_pcidas.o.flags
? comedi/drivers/.comedi_parport.o.flags
? comedi/drivers/.daqboard2000.o.flags
? comedi/drivers/.das08.o.flags
? comedi/drivers/.das16.o.flags
? comedi/drivers/.das1600.o.flags
? comedi/drivers/.das1800.o.flags
? comedi/drivers/.das6402.o.flags
? comedi/drivers/.das800.o.flags
? comedi/drivers/.dt2801.o.flags
? comedi/drivers/.dt2811.o.flags
? comedi/drivers/.dt2814.o.flags
? comedi/drivers/.dt2815.o.flags
? comedi/drivers/.dt2817.o.flags
? comedi/drivers/.dt282x.o.flags
? comedi/drivers/.dt3000.o.flags
? comedi/drivers/.fl512.o.flags
? comedi/drivers/.ii_pci20kc.o.flags
? comedi/drivers/.multiq3.o.flags
? comedi/drivers/.ni_atmio.o.flags
? comedi/drivers/.ni_atmio16d.o.flags
? comedi/drivers/patch1
? comedi/drivers/.ni_mio_cs.o.flags
? comedi/drivers/.ni_pcidio.o.flags
? comedi/drivers/patch
? comedi/drivers/.ni_pcimio.o.flags
? comedi/drivers/.pcl711.o.flags
? comedi/drivers/.pcl724.o.flags
? comedi/drivers/.pcl725.o.flags
? comedi/drivers/.pcl726.o.flags
? comedi/drivers/.pcl812.o.flags
? comedi/drivers/.pcl818.o.flags
? comedi/drivers/.poc.o.flags
? comedi/drivers/.rti800.o.flags
? comedi/drivers/.rti802.o.flags
? comedi/drivers/.skel.o.flags
? comedi/drivers/ack
? comedi/drivers/.adv_pci1710.o.flags
? comedi/drivers/.quatech_daqp_cs.o.flags
? comedi/kcomedilib/.depend
? comedi/kcomedilib/.data.o.flags
? comedi/kcomedilib/.dio.o.flags
? comedi/kcomedilib/.kcomedilib_main.o.flags
? comedi/kcomedilib/.ksyms.o.flags
? comedi/realtime/.depend
? comedi/rt_pend_tq/.depend
? comedi/rt_pend_tq/.rt_pend_tq.o.flags
? include/modbuild/config.h
? include/modbuild/version.h
? scripts/mkdep
Index: Makefile.modbuild
===================================================================
RCS file: /var/cvs/comedi/Makefile.modbuild,v
retrieving revision 1.13
diff -u -r1.13 Makefile.modbuild
--- Makefile.modbuild	2001/03/01 05:03:23	1.13
+++ Makefile.modbuild	2001/04/03 23:48:29
@@ -57,6 +57,7 @@
 ifdef RTAIDIR
 CFLAGS += -I $(TOPDIR)/include/rtai
 endif
+CFLAGS += -I /usr/realtime/include
 
 CONFIG_SHELL := sh
 
Index: scripts/dep.rtai
===================================================================
RCS file: /var/cvs/comedi/scripts/dep.rtai,v
retrieving revision 1.3
diff -u -r1.3 dep.rtai
--- scripts/dep.rtai	2001/01/09 19:18:10	1.3
+++ scripts/dep.rtai	2001/04/03 23:48:30
@@ -5,11 +5,19 @@
 
 	if [ "$RTAIDIR" = "" ]
 	then
-		echo -n "Enter location of RTAI source tree [/usr/src/rtai]: "
-		read RTAIDIR
-		: ${RTAIDIR:=/usr/src/rtai}
+		if [ -d /usr/realtime/include/rtai ];then
+			echo "You seem to have RTAI headers installed.  We'll use those..."
+		else
+			echo -n "Enter location of RTAI source tree [/usr/src/rtai]: "
+			read RTAIDIR
+			: ${RTAIDIR:=/usr/src/rtai}
+		fi
 	fi
-	echo RTAIDIR=$RTAIDIR >>.sourcedirs
-	(cd include;rm -f rtai;ln -s $RTAIDIR/include rtai)
+	if [ "$RTAIDIR" = "" ];then
+		(cd include;rm -f rtai;ln -s /usr/realtime/include/rtai rtai)
+	else
+		echo RTAIDIR=$RTAIDIR >>.sourcedirs
+		(cd include;rm -f rtai;ln -s $RTAIDIR/include rtai)
+	fi
 fi
 
